FV3/gfsphysics/GFS_layer/GFS_diagnostics.F90

```diff
@@ -3812,6 +3812,45 @@ subroutine GFS_externaldiag_populate (ExtDiag, Model, Statein, Stateout, Sfcprop
     ExtDiag(idx)%data(nb)%var2 => IntDiag(nb)%q_dt_int(:,5)
   enddo
 
+  idx = idx + 1
+  ExtDiag(idx)%axes = 2
+  ExtDiag(idx)%name = 'tsfc_via_diagnostics_manager'
+  ExtDiag(idx)%desc = 'surface temperature'
+  ExtDiag(idx)%unit = 'K'
+  ExtDiag(idx)%mod_name = 'gfs_sfc'
+  ExtDiag(idx)%coarse_graining_method = 'area_weighted'
+  ExtDiag(idx)%diag_manager_controlled = .TRUE.
+  allocate (ExtDiag(idx)%data(nblks))
+  do nb = 1,nblks
```

Comment:
Not sure what `blk` is? Bulk?

Reply:
I think it stands for "block," and corresponds to how groups of columns are partitioned in the physics code (a lot of the variable names in here are things we've inherited for better or worse).